lib/reporters/cli/index.js

```diff
@@ -24,6 +25,19 @@ colors.setTheme({
     error: 'red'
 });
 
+extractSNR = function (executions) {
+    var snr = {};
+
+    _.isArray(executions) && executions.forEach(function (execution) {
+        _.has(_.get(execution, 'result.return'), 'nextRequest') && (
+            (snr.defined = true),
+            (snr.value = execution.result.return.nextRequest)
+        );
+    });
+
+    return snr;
```

Comment:
It's cleaner to return the matched value directly. Use that in the condition below as well.